.lintstagedrc.json

```diff
@@ -2,7 +2,7 @@
   "*.{css,scss}": [
     "stylelint"
   ],
-  "*.{js,jsx,ts,tsx}": [
+  "(!libs/react/**/*)*.{js,jsx,ts,tsx}": [
```

Comment:
Not your fault, but this whole file can be removed. I fixed this issue in parent branches.